### PR TITLE
S01 us12

### PR DESCRIPTION
**Pull request by Halicksse**
I added the navlink to all the links in the footer.

### CODE REVIEW

**Review by Jubdc — APPROVED** [review]

**Review by YOUNS28100 — APPROVED** [review]

File: frontend/src/components/CurrentRead.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import "../styles/currentread.css";
```
Comment (ioayoub) [inline frontend/src/components/CurrentRead.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/About.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/pages/Footer/About.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/Cgv.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/pages/Footer/Cgv.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/Contact.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/pages/Footer/Contact.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/Delivery.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/pages/Footer/Delivery.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/Faq.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/pages/Footer/Faq.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/General.jsx [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/pages/Footer/General.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/Legal.jsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
+function Legal() {
```
Comment (ioayoub) [inline frontend/src/pages/Footer/Legal.jsx:1]: Empty components are not allowed.

File: frontend/src/pages/Footer/Privacy.jsx [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/pages/Footer/Privacy.jsx:1]: Empty components are not allowed.

File: frontend/src/styles/footer.css [hunk 10]
```diff
@@ -6,11 +6,11 @@
 .footer {
   display: flex;
   justify-content: space-around;
-  box-sizing: border-box;
+  /* box-sizing: border-box; */
```
Comment (ioayoub) [inline frontend/src/styles/footer.css:9]: 👀

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Pull request closed without merging**